e2e/scripts/execute-shell-commands.js

```diff
@@ -34,10 +34,10 @@ const runGeneratorWithResponses = (cmd, cliResponses = []) => {
     child.stderr.on("data", (err) => {
       // Lerna warnings are also seen as errors but we want to continue in those cases
       // We exit the process if something breaks after the generator is actually running.
-      if (isGenratorRunning) {
-        console.error(err);
-        reject(err);
-      }
+      // if (isGenratorRunning) {
+      //   console.error(err);
+      //   reject(err);
+      // }
```

Comment:
stderr is triggered during `npm warn`, for example

![CleanShot 2024-05-28 at 23 12 32](https://github.com/SalesforceCommerceCloud/pwa-kit/assets/10948652/eb7dc59a-e89c-405f-9b92-134b08a12b39)

deprecated dependencies will trigger the commented code and cause e2e test to fail.